R/data.R

```diff
@@ -0,0 +1,205 @@
+# unanchored datasets ------
+
+#' Patient data from single arm study
+#' @format a data frame with 500 rows and 8 columns:
+#'   \describe{
+#'     \item{USUBJID}{Unique subject identifiers for patients.}
+#'     \item{ARM}{Assigned treatment arm.}
+#'     \item{AGE}{Age in years at baseline.}
+#'     \item{SEX}{Sex of patient recorded as character `"Male"`/`"Female"`.}
+#'     \item{SMOKE}{Smoking status at baseline as integer `0`/`1`.}
+#'     \item{ECOG0}{ECOG score at baseline as integer `0`/`1`.}
+#'     \item{N_PR_THER}{Number of prior therapies received as integer `1, 2, 3, 4`.}
+#'     \item{SEX_MALE}{Indicator of `SEX == "Male"` as numeric `1`/`0`.}
+#'
+#'   }
+#' @keywords dataset
+#' @family unanchored datasets
+"adsl_sat"
+
+#' Survival data from single arm trial
+#' @format A data frame with 500 rows and 10 columns:
+#'   \describe{
+#'     \item{USUBJID}{Unique subject identifiers for patients.}
+#'     \item{ARM}{Assigned treatment arm, `"A"`.}
+#'     \item{EVNT}{Event indicator `0`/`1`.}
+#'     \item{AVAL}{Analysis value which in this dataset overall survival time in days.}
+#'     \item{AVALU}{Unit of `AVAL`.}
+#'     \item{PARAMCD}{Paramater code of `AVAL`, `"OS"`.}
+#'     \item{PARAM}{Parameter name of `AVAL`, `"Overall Survival`.}
+#'     \item{CNSR}{Censoring indicator `0`/`1`.}
+#'     \item{TIME}{Survival time in days.}
+#'     \item{EVENT}{Event indicator `0`/`1`.}
```

Comment:
EVENT and EVNT are same is that okay? I guess so?

Reply:
Remove `EVNT` and check code works